docs/reference/configuration/queue.rst

```diff
@@ -183,10 +183,11 @@ The following is a list of available LSF configuration options:
   LSF uses resource requirements to select hosts for remote execution and
   job execution. Resource requirement strings can be simple (applying to the
   entire job) or compound (applying to the specified number of slots).
+  The set unit for memory booking is MB, so all values here will be in MB.
```

Comment:
We need to be careful here, because the default is installation dependent. LSF9 has a default at KB, while our onpremise data cluster has reconfigured the default to MB. 

(When we progress to LSF10 (soonish), there will be support for units, and then our documentation becomes correct)

Reply:
As long as the documentation is valid at the time of the user reading this, then that should be fine. Is there a point in future proofing it? When when we go over to LSF10, we have to update the docs again anyways.

Reply:
The documentation in this PR will not be valid for anyone using Ert outside of Equinors premises..

Reply:
Is anyone using it outside of Equinor's premises? I can mention that this is Equinor specific, but should I also say how it is normally, outside of Equinor's premises?

Reply:
Ert is public so we cannot make the assumption that it is not used outside.